server/src/main/java/org/candlepin/resource/StatusResource.java

```diff
@@ -154,22 +156,30 @@ public StatusDTO status() {
         CandlepinCapabilities caps = CandlepinCapabilities.getCapabilities();
 
         RulesSourceEnum rulesSource = jsProvider.getRulesSource();
+        StatusDTO status = null;
 
-        StatusDTO status = new StatusDTO()
-            .setResult(good)
-            .setVersion(version)
-            .setRelease(release)
-            .setStandalone(standalone)
-            .setRulesVersion(jsProvider.getRulesVersion())
-            .setRulesSource(rulesSource != null ? rulesSource.toString() : null)
-            .setMode(mode != null ? mode.toString() : null)
-            .setModeReason(modeChangeReason != null ? modeChangeReason.toString() : null)
-            .setModeChangeTime(modeChange.getChangeTime())
-            .setManagerCapabilities(caps)
-            .setTimeUTC(new Date())
-            .setResource(resource)
-            .setAuthUrl(authUrl)
-            .setRealm(realm);
+        if (adapterConfig != null) {
+            KeycloakStatusDTO keycloakstatus = new KeycloakStatusDTO();
```

Comment:
You can avoid the redeclaration and extra reassignment here by using invocation chaining:

```
if (adapterConfig != null) {
    status = new KeycloakStatusDTO()
        .setKeycloakResource(...)
        .setKeycloakAuthUrl(...)
        ...
}
```